app/views/step_by_step_pages/internal_change_notes.html.erb

```diff
@@ -45,25 +44,27 @@
       } %>
     <% end %>
 
-    <% @step_by_step_page.internal_change_notes.each do |entry| %>
-      <div class="app-c-timeline-entry">
-        <% if displayed_edition_number != entry.edition_number %>
```

Comment:
As this check always evaluated to true, and therefore was never needed, can you fix up it's removal to the commit it was added in.